### ISSUE TITLE
Patterns Chapter

### ISSUE THREAD

**Issue by lennart** [tidalcycles/tidalcycles.github.io#5]
Currently this chapter lacks a bit of structure. For me the purpose is not really clear.

Mainly this should give an explanation of Tidal's embedded DSL for patterns, and continuous patterns like e.g. `sinewave`.

In contrast to "Getting started" this should go more into detail, while keeping things visualized with examples instead of being a pure function listing (as seen in the Reference)

**Comment by kindohm** [comment]
I've also received a lot of feedback about how the docs don't really explain concepts very well. _Creating Patterns_ explains the patterns DSL, and _Reference_ documents each function, but we don't have a good chapter for explaining larger concepts about strategies for using everything together.

**Comment by lennart** [comment]
right, this is also reflected in the responses to your new video tutorial for tidal 0.6 (http://lurk.org/groups/tidal/messages/topic/2UoxitwI3GzXNij4IWmHtA), strategies and concepts also seem to be words, that mean something for software developers, but could also be repelling for non-developers…

I'll be adding a separate issue for the _soon to be created_ chapter
